client/my-sites/sidebar-unified/style.scss

```diff
@@ -41,167 +41,196 @@ $font-size: rem( 14px );
 	}
 }
 
-.is-nav-unification {
-	// client/layout/style.scss
-	.layout__content {
-		min-height: 101vh; // Hack to give JS the chance to trigger the scroll event when the content is short. JS code: client/layout/utils.ts:36.
+// client/layout/style.scss
```

Comment:
Changes here look much more bigger than they really are – I just removed the `.is-nav-unification {` wrapper and outdented the inner selectors in 1 tab.

Reply:
They are way easier to review by using the unified diff view with whitespaces hidden:
<img width="269" alt="Screen Shot 2022-03-25 at 11 04 35" src="https://user-images.githubusercontent.com/1233880/160100383-1c5ef71d-75ab-4377-b07d-94e3a17b4f51.png">